CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
I remember having to upgrade the minimum required cmake at some point. Version 3.2 is from 2015, is this requirement an issue?